README.md

```diff
@@ -1094,6 +1095,89 @@ Server: spray-can/1.3.3
 }
 ```
 
+## GET /workflows/:version/:id/metadata
```

Comment:
Fine by me but I don't really care what it is specifically, just that the call specific part would need to be changed next sprint so might as well do it now